test_programs/compile_success_empty/trait_generics/src/main.nr

```diff
@@ -8,21 +7,21 @@ fn main() {
     assert_eq(15, sum_static(Data { a: 5, b: 10 }));
 }
 
-fn foo<T, U>(x: T, u: U) where T: Into<U>, U: Eq {
+fn foo<T, U>(x: T, u: U) where T: MyInto<U>, U: Eq {
     assert(x.into() == u);
 }
 
-trait Into<T> {
+trait MyInto<T> {
```

Comment:
Got confused at first -- this rename is needed because Into is not in the prelude